parsing/parser.mly

```diff
@@ -2960,7 +2962,7 @@ sig_exception_declaration:
   attrs2 = attributes
   attrs = post_item_attributes
     { let args, res = args_res in
-      let loc = make_loc $sloc in
+      let loc = make_loc ($startpos, $endpos(attrs2)) in
```

Comment:
Again, is it possible to slightly rewrite the rule to have this be the location of some symbol?